assets/locales/en.json

```diff
@@ -39,6 +39,7 @@
     "no_success": "Cancelled",
     "disclosure_success": "Success, your data has been shared",
     "issuance_success": "Success, the data has been added",
+    "signature_success": "Success, the message has been signed with your data",
```

Comment:
I know the task specified this as the translation, but it differs in meaning from the Dutch translation. Maybe change it to 'Success, you have signed the message' to match the Dutch translation?